schema.sql

```diff
@@ -0,0 +1,20 @@
+DROP DATABASE IF EXISTS prodimgs;
+
+CREATE DATABASE prodimgs;
+
+USE prodimgs;
+
+CREATE TABLE products (
+  id int NOT NULL AUTO_INCREMENT,
+  name varchar(40) NOT NULL,
+  price int NOT NULL,
+  category varchar(30) NOT NULL,
+  product_url varchar(100) NOT NULL,
+  main_img varchar(100) NOT NULL,
```

Comment:
Oh, one more thing, just a suggestion -  I see you've put NOT NULL for the front/back/bottom/top/right views - I'm guessing these are for the images? Perhaps removing 'NOT NULL' will give you more flexibility when inserting records into the db (in case we don't have an image link for the front/back/bottom/top right)